pkgs/star-citizen/default.nix

```diff
@@ -0,0 +1,106 @@
+{
+  lib,
+  makeDesktopItem,
+  symlinkJoin,
+  writeShellScriptBin,
+  gamemode,
+  winetricks,
+  wine,
+  dxvk,
+  wineFlags ? "",
+  pname ? "star-citizen",
+  location ? "$HOME/Games/star-citizen",
+  tricks ? ["arial" "vcrun2019" "win10"],
+  preCommands ? "",
+  postCommands ? "",
+}: let
+  version = "1.6.7";
+  src = builtins.fetchurl rec {
```

Comment:
Maybe we can use `pkgs.fetchurl` here, so that it supports SRI `hash` instead of `sha256`.

Reply:
I truthfully don’t know anything about about sri hashes, a quick google search lead me to RFC 0131

Reply:
Yes. It was put in place so that transitions to another hash format would be seamless.